packages/block-library/src/form-input/edit.js

```diff
@@ -63,8 +63,7 @@ function InputFieldBlock( { attributes, setAttributes, className } ) {
 			) }
 			<InspectorControls group="advanced">
 				<TextControl
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
Enable "Form and input blocks" in Gutenberg Experimental settings. Insert Form block. Click on one of the input blocks.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/081250bb-2099-4cb1-b9cc-563871f7f487" alt="FormInput, before" width="271">|<img src="https://github.com/user-attachments/assets/1fdb1bae-ddac-45fc-9288-e1a286c2200c" alt="FormInput, after" width="271">|